src/wp-includes/block-supports/dimensions.php

```diff
@@ -63,23 +63,6 @@ function wp_apply_dimensions_support( $block_type, $block_attributes ) { // phpc
 	return empty( $styles ) ? array() : array( 'style' => implode( ' ', $styles ) );
 }
 
-/**
- * Checks whether serialization of the current block's dimensions properties
- * should occur.
- *
- * @since 5.9.0
- * @access private
- *
- * @param WP_Block_type $block_type Block type.
- * @return bool Whether to serialize spacing support styles & classes.
- */
-function wp_skip_dimensions_serialization( $block_type ) {
```

Comment:
@peterwilsoncc or @hellofromtonya, should we keep functions like this one with no code and depreciation notice for backward compatibility or is it fine to remove as they were marked as private?

Reply:
The practice is to move the functions to deprecated and throw a warning, for example:

https://github.com/WordPress/wordpress-develop/blob/6c270d0d176285b94e018f86187a25f13a7a4ebf/src/wp-includes/deprecated.php#L3373-L3386

Unfortunately plugin authors have a habit of ignoring private access statements. To be fair, some of this is self inflicted by things such as `WP_List_Table` still being marked as private.

Reply:
Thanks folks! If we decide it's a good idea to reinstate them, I've thrown up a PR here: https://github.com/WordPress/wordpress-develop/pull/2522